packages/forms-web-app/src/controllers/eligibility/granted-or-refused-permission.js

```diff
@@ -38,36 +58,24 @@ exports.postGrantedOrRefusedPermission = async (req, res) => {
     return;
   }
 
-  let isPlanningPermissionRefused = null;
-
-  if (validHouseholderPlanningPermissionStatusOptions.includes(planningPermissionStatus)) {
-    isPlanningPermissionRefused =
-      planningPermissionStatus.toLowerCase() === ELIGIBILITY.PLANNING_PERMISSION_STATUS.REFUSED;
-  }
-
   try {
     req.session.appeal = await createOrUpdateAppeal({
       ...appeal,
       eligibility: {
         ...appeal.eligibility,
-        planningPermissionStatus,
+        planningPermissionStatus: selectedPermissionStatus,
       },
     });
   } catch (e) {
     logger.error(e);
 
-    res.render(VIEW.ELIGIBILITY.GRANTED_REFUSED_PERMISSION, {
+    res.render(forwardPage(selectedPermissionStatus), {
       appeal,
       errors,
       errorSummary: [{ text: e.toString(), href: '#' }],
     });
     return;
   }
 
-  if (!isPlanningPermissionRefused) {
-    res.redirect(`/${VIEW.ELIGIBILITY.GRANTED_REFUSED_PERMISSION_OUT}`);
-    return;
-  }
-
-  res.redirect(`/${VIEW.ELIGIBILITY.DECISION_DATE}`);
+  res.redirect(`/${forwardPage(selectedPermissionStatus)}`);
```

Comment:
Can we add an additional unit test to test the redirect to the `/no-decision` page please?

Reply:
The one you added in the controller tests that the correct page is loaded for a GET request. 

The one I'm referring to tests that the `/eligibility/no-decision` page is loaded if the 'I have not received a decision' option is selected on the `/eligibility/granted-or-refused-permission` page.

It would be an equivalent test to https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/packages/forms-web-app/__tests__/unit/controllers/eligibility/granted-or-refused-permission.test.js#L123

Reply:
Good catch. Resolved all the review comments